packages/react-scripts/template-typescript/src/App.tsx

```diff
@@ -1,28 +1,26 @@
-import React, { Component } from 'react';
+import * as React from 'react';
```

Comment:
Wouldn't `import React, { FC } from 'react';` be more consistent in spirit to the current class-based version?

Reply:
@denisw In [TypeScript Doc](https://www.typescriptlang.org/docs/handbook/react-&-webpack.html#write-some-code) it write:

```ts
import * as React from "react";
```

Should we use named export rather than default export?

Reply:
Yes, `import * as React` is what you would need to do by default, but create-react-app's TypeScript configuration has `allowSyntheticDefaultImports` option enabled. From the [documentation](https://www.typescriptlang.org/docs/handbook/compiler-options.html):

> Allow default imports from modules with no default export. This does not affect code emit, just typechecking.